cypress/integration/login.js

```diff
@@ -20,5 +20,6 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('.calendar--today').contains(15)
```

Comment:
This test will fail tomorrow, maybe we can do something that would prevent that? e.g a hint if you'll get stuck :) https://www.w3schools.com/jsref/jsref_getday.asp

Reply:
https://www.w3schools.com/jsref/jsref_getdate.asp